CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Changelog
-
+test
```

Comment:
revert before merging?

Reply:
yep - still debugging since it ran all the CI despite the new `ignore-paths` 😞